### PR TITLE
NPEC

### PR DESCRIPTION
**Pull request by Joedirte1917**
Added a preliminary overview of the purpose, structure, content, and responsibility of the NPEC.

### CODE REVIEW

File: Committees/NPEC.adoc [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+== Purpose
+
+Political education of PLTCom members is the sole mission of this committee. It will provide all members a course in capitalism first and scientific socialism second followed by a course on party work. It will also maintain a comprehensive digital library of marxist political material as well as a physical library as possible. 
+
+== Structure
+- Committee Chair
+- Librarian
+- Teacher(s)
+
+=== Courses
```
Comment (1bigchonk) [inline Committees/NPEC.adoc]: I think the courses should be in a separate file. In my head, this file is to outline the duties of the committee as a reference to people who want to join a committee. We absolutely should construct a course catalog. I have a draft syllabus in a different repo we could easily copy and paste over for this repo.

File: Committees/NPEC.adoc [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+- Lenin "The Right of Nations to Self-Determination"
+
+- Fanon "Wretched of the Earth"
+
+- Haywood "For A Revolutionary Position on the Negro Question"
+
+== Requirements
+
+Everyone on the committee **must** be able to prove comprehension of _any_ material they submit. Anyone considered for the committee must also be able to demonstrate an education in the works above either through PLT-Com or elsewhere. People should only teach what they are themselves profficient in. As well, this committee should work with lower level committees to provide assistance and guidance for their local educational efforts.
```
Comment (1bigchonk) [inline Committees/NPEC.adoc:29]: As a bit of an aside, what do you think the base requirements for each members' political education be? I was thinking a simple, one off course so people could get started quickly. That may not be sufficient, and based on what you've outlined here it should be 3 courses? How much work can members do before they've completed all required coursework? Are we going to lose people by not letting them participate in work before they log all of these classes?

**Comment by 1bigchonk** [comment]
Formatting looks good, I didn't see any obvious grammatical or spelling errors, things were easy to read and understand.

File: Committees/NPEC.adoc (hunk 1 above)
Comment (Joedirte1917) [reply Committees/NPEC.adoc]: Ah ok yeah that would be fine to move it there.

File: Committees/NPEC.adoc (hunk 2 above)
Comment (1bigchonk) [inline Committees/NPEC.adoc:29]: How frequently does this committee meet? How are we proving comprehension?

Comment (Joedirte1917) [reply Committees/NPEC.adoc:29]: I would say, they can participate, but if they want to hold like a chapter leadership position or national one they need to at least have all the 101s done.

**Review by 1bigchonk — CHANGES_REQUESTED** [review]
See comments

File: Committees/NPEC.adoc (hunk 2 above)
Comment (Joedirte1917) [reply Committees/NPEC.adoc:29]: And we could say that only full members can hold those positions and make the distinction that you become a full member when you complete the 101s

Comment (Joedirte1917) [reply Committees/NPEC.adoc:29]: They should meet at a minimum of 4 times a month. The CC should compile a test for each of the courses (tho, only when they are going to be run) and use that as a gauge.

**Review by 1bigchonk — APPROVED** [review]

**Pull request merged**